demo/client/src/ml-models/deploy-model-node.js

```diff
@@ -10,7 +10,9 @@ const { convertData, convertNum } = require('../float-utils-node')
 
 const _toFloat = 1E9
 
-async function deployDensePerceptron(model, web3, toFloat) {
+async function deployDensePerceptron(model, web3, options){
+    const {toFloat, initialChunkSize = 200, chunkSize = 250,
```

Comment:
The defaults in each method should match the defaults that were there before.

Reply:
should I change all weightChunkSize with chunk size parameter?

Reply:
if So, what is the purpose of initialChunkSize in options? where to use that?

Reply:
`weightChunkSize` should usually become `chunkSize`. For some of the methods, I didn't before with initial chunk sizes but you can change the appropriate `weightChunkSize` or `chunkSize` to use `initialChunkSize` for when the contract is created. If there was no initial chunk size in the method, then you can just default the value to say as the chunk size value for that method.

Reply:
Kindly check it now